osbs/api.py

```diff
@@ -799,6 +830,81 @@ def create_build(self, **kwargs):
         """
         return self._do_create_prod_build(**kwargs)
 
+    @osbsapi
+    def create_source_container_build(
+        self,
+        sources_for_koji_build_nvr=None,
+        outer_template=None,
+        arrangement_version=None,
+        scratch=None,
+        user=None,
+        platform=None,
+        koji_task_id=None,
+        reactor_config_override=None,
+        target=None,
+    ):
+        """
+        Take input args, create build request and submit the source image build
+
+        :return: instance of BuildRequest
+        """
+        build_request = self.get_source_container_build_request(
+            outer_template=outer_template or ORCHESTRATOR_SOURCES_OUTER_TEMPLATE,
+            arrangement_version=arrangement_version
+        )
+
+        if not sources_for_koji_build_nvr:
+            raise OsbsValidationException(
+                "required argument 'sources_for_koji_build_nvr' can't be None"
+            )
+
+        name, version, release = sources_for_koji_build_nvr.split('-', 3)
+
+        build_request.set_params(
+            arrangement_version=arrangement_version,
+            component=name,  # TODO check this
```

Comment:
:eyes: 

is this ok?

Reply:
Hmm, no.

Maybe `create_source_container_build` should take `component` as a parameter, as long as it's easy enough for koji-containerbuild to get the right value and pass it in(?).

Reply:
What should be used for component? with image builds It is defined by users in Dockerfile. Should we ask users to pass component?

Reply:
Agreed offline: osbs-client will not require component, it will be responsibility of atomic-reactor to get it from original build.

I'd keep it as is and remove component in separate PR